database/tests/factories/core.py

```diff
@@ -71,9 +71,11 @@ class Meta:
 
     owner = factory.SubFactory(OwnerFactory)
     bot = None
-    updatestamp = factory.LazyAttribute(lambda o: datetime.now())
+    updatestamp = factory.LazyAttribute(lambda o: datetime.now(tz=timezone.utc))
     languages = []
-    languages_last_updated = factory.LazyAttribute(lambda o: datetime.now())
+    languages_last_updated = factory.LazyAttribute(
+        lambda o: datetime.now(tz=timezone.utc)
```

Comment:
Sentry uses `django.utils.timezone.now()` for these. Would that make sense here as well?

Reply:
Hmm, these are factories to create the SQLAlchemy models, so I don't think using a Django Utils is appropriate.